node/src/Pos/Client/Txp/Balances.hs

```diff
@@ -16,20 +16,21 @@ import           Pos.Core             (Address (..), Coin, IsBootstrapEraAddr (.
                                        isRedeemAddress, makePubKeyAddress)
 import           Pos.Crypto           (PublicKey)
 import           Pos.DB               (MonadDBRead)
-import           Pos.Txp              (MonadTxpMem, Utxo, addrBelongsToSet,
-                                       getUtxoModifier)
+import           Pos.Txp              (Utxo, UtxoModifier, addrBelongsToSet)
 import qualified Pos.Txp.DB           as DB
 import           Pos.Txp.Toil.Utxo    (getTotalCoinsInUtxo)
 import qualified Pos.Util.Modifier    as MM
 import           Pos.Wallet.Web.State (WalletSnapshot)
 import qualified Pos.Wallet.Web.State as WS
 
-getOwnUtxos :: (MonadIO m, MonadDBRead m, MonadTxpMem ext ctx m)
-            => WalletSnapshot -> [Address] -> m Utxo
-getOwnUtxos ws addrs = do
+getOwnUtxos :: (MonadIO m, MonadDBRead m)
+            => WalletSnapshot
+            -> UtxoModifier
+            -> [Address]
+            -> m Utxo
+getOwnUtxos ws updates addrs = do
```

Comment:
Nice one. It's now explicit from the type signature that "if you give me the wallet snapshot and any modification I need to apply to the Utxo, I will give you back the Utxo". Glad to see `MonadTxpMem` gone!